content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImport.class/__class__.yaml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
@tinaafitz ok, 1) and 2) seems pretty straightforward (I assume the states to be of type `string`) 
but regarding 3) what is exactly necessary? add 3 *methods* for each state of the state machine (with the new pre and post that would total to 8*3=24 new methods)?